### DISCUSSION TITLE
Issues with Docker Desktop 4.17.1

### DISCUSSION THREAD

**Discussion by jkovacs-dev** [AbdBarho/stable-diffusion-webui-docker#372] [General]
For anyone running Docker Desktop on Windows using an NVIDIA GPU, just be aware that there seems to be an issue in 4.17.1 that causes containers using the GPU (like the ones in this repo) to freeze on startup.  Downgrading to 4.17.0 seems to fix it.  Just posting it here so others don't waste the hours I did pulling their hair out over this... 

Discussion of the issue is [here](https://github.com/docker/for-win/issues/13100).

**Comment by AbdBarho** [comment]
Thank you! relevant issue: https://github.com/AbdBarho/stable-diffusion-webui-docker/issues/369
